src/main/java/hudson/plugins/active_directory/ActiveDirectoryAuthenticationProvider.java

```diff
@@ -99,7 +115,11 @@ static String dnToLdapUrl(String dn) {
         return "LDAP://"+dn.replace("/","\\/");
     }
 
-    protected UserDetails retrieveUser(String username, UsernamePasswordAuthenticationToken authentication) throws AuthenticationException {
+    /**
+     * Perform a look for and retrieve the UserDetails
+     * @return UserDetails
+     */
+    protected UserDetails computeUser(String username, UsernamePasswordAuthenticationToken authentication) throws AuthenticationException {
```

Comment:
nit: compute is really the wrong thing as it's not really computing - this is getting something from LDAP so `retreiveUserFromServer` may be more intuitive.